src/main/webapp/resources/js/pages/projects/samples-metadata-import2/components/SampleMetadataImportSteps.jsx

```diff
@@ -19,4 +20,9 @@ export function SampleMetadataImportSteps({ currentStep, currentStatus }) {
       <Step title={i18n("SampleMetadataImportSteps.step4")} />
     </Steps>
   );
-}
\ No newline at end of file
+}
+
+SampleMetadataImportSteps.propTypes = {
```

Comment:
So funny thing, can you actually remove these, Deep and I are going to go through and remove all of these shortly.  They really do not provide that much benifit for us and can be a pain.

Reply:
Please see 55d009a5e7e0ec7bc5d7f3211d216a359100d701.